### ISSUE TITLE
iOS Compatibility

### ISSUE THREAD

**Issue by tslater** [mhakman/osx-cocoa-odbc#2]
Any idea how difficult this would be to port over to iOS?

**Comment by jfosback** [comment]
I don't think it would be possible to port this to iOS at all.  It uses iODBC dylibs, and I seriously doubt that will get ported to iOS.  You would also need ODBC drivers to talk to whatever database you need to on iOS, another hurdle.

I got it working on OS X, and it works great.  That said, it may be worth trolling the iODBC web site to see if they have any iOS plans, but you'd still need native iOS database drivers from whatever vendor you use.
